### PR TITLE
Added largest of three numbers

### PR DESCRIPTION
**Pull request by RAHULANUP**
Added largest of three numbers in java....

### CODE REVIEW

**Comment by Dhruv031003** [comment]
@RAHULANUP 
Readme is assigned to someone else please remove it.

**Comment by Dhruv031003** [comment]
There is no conditon to handle the event when all the nos are equal.

**Comment by RAHULANUP** [comment]
@Dhruv031003 deleted the README...

**Comment by RAHULANUP** [comment]
@Dhruv031003 Added condition for equal numbers as well...

File: Largestofthreenum/largestofthreenum.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Scanner sc = new Scanner(System.in);  
+//reading input from the user  
+System.out.println("Enter number 1");  
+n1 = sc.nextInt();  
+System.out.println("Enter number 2");  
+n2 = sc.nextInt();  
+System.out.println("Enter number 3");  
+n3 = sc.nextInt();
+if(n1==n2 && n2==n3){
+  System.out.println("ALL THE NUMBERS ARE EQUAL);
```
Comment (Dhruv031003) [inline Largestofthreenum/largestofthreenum.java:17]: line 17 quote missing

**Comment by Dhruv031003** [comment]
@RAHULANUP 
Code should show nos are equal or the one which is largest no both things

**Comment by Dhruv031003** [comment]
> @RAHULANUP Code should show nos are equal or the one which is largest not both.

@RAHULANUP Fix this

**Review by alok-96 — COMMENTED** [review]
Showing the incorrect output when a=b=c 
Printing both the lines 17 and 23.

**Comment by RAHULANUP** [comment]
@alok-96 updated...

**Pull request merged**